archunit/src/test/java/com/tngtech/archunit/testutil/Assertions.java

```diff
@@ -228,8 +227,4 @@ public static CodeUnitAccessAssertion assertThatCall(JavaConstructorCall call) {
     public static TryCatchBlockAssertion assertThatTryCatchBlock(TryCatchBlock tryCatchBlock) {
         return new TryCatchBlockAssertion(tryCatchBlock);
     }
-
-    public static TryCatchBlocksAssertion assertThatTryCatchBlocks(Set<TryCatchBlock> tryCatchBlocks) {
```

Comment:
This change is currently in the commit `make processing of raw try-catch-blocks consistent with other imports`, but I think that it would better fit in `remove obsolete class `TryCatchBlocksAssertion`.